packages/rocketchat-livechat/server/api/lib/livechat.js

```diff
@@ -0,0 +1,98 @@
+import _ from 'underscore';
+import LivechatVisitors from '../../models/LivechatVisitors';
+
+export const online = RocketChat.models.Users.findOnlineAgents().count() > 0;
```

Comment:
this should be a function as well because it currently runs only at startup and will always reflect the initial server state..

changing to a function will make it be evaluated every time it's called, which is the desired behavior.